src/lib/group.ts

```diff
@@ -20,6 +23,25 @@ class Group {
     return new Group({ x: Pallas.one.x, y: Pallas.one.y });
   }
 
+  /**
+   * Unique representation of the `zero` element of the Group (the identity element of addition in this Group).
+   *
+   * **Note**: The `zero` element is represented as `(1, 1)`.
+   *
+   * ```typescript
+   * // g + -g = 0
+   * g.add(g.neg()).assertEquals(zero);
+   * // g + 0 = g
+   * g.add(zero).assertEquals(g);
+   * ```
+   */
+  static get zero() {
+    return new Group({
```

Comment:
`zero` is `(1, 1)`, similar to the projective zero in the crypto implementation - `(0, 0)` may be a little more obvious but would have to make sure the native crypto code handles it the same way